src/server/node_manager.c

```diff
@@ -1104,25 +1104,16 @@ shallow_vnode_dup(struct pbsnode *vnode)
 	/* 
 	 * Allocate and initialize vnode_o, then copy vnode elements into vnode_o
 	 */
-	if ((vnode_dup = malloc(sizeof(struct pbsnode)))) {
-		if (initialize_pbsnode(vnode_dup, strdup(vnode->nd_name), NTYPE_PBS) != PBSE_NONE) {
-			log_err(PBSE_INTERNAL, __func__, "vnode_dup initialization failed");
-			free_pnode(vnode_dup);
-			return NULL;
-		}
-	} else {
+	if ((vnode_dup = calloc(1, sizeof(struct pbsnode))) == NULL) {
```

Comment:
you removed the call to initialize_pbsnode altogether, was that intentional?

Reply:
Yes, these initialized values are getting freed and re-assigned.